### DISCUSSION TITLE
Custom CSS styling can't apply when downloading PNG

### DISCUSSION THREAD

**Discussion by ghnmqdtg** [apexcharts/apexcharts.js#2867] [Q&A]
Hi! Recently I've been developed an analysis app for scientific research with Apexchart to plot some raw data.

In this project, I create custom charts with Apexchart options and CSS files, such as font-family(Roboto), tooltip title colour, tooltip marker radius, and width of legends. Everything looks great as I expected it to be in my app. But, when I download charts as PNG files, all the custom stylings run unavailable.

I also tried using html2canvas to output the chart image to PDF file by jsPDF, but still got the same issue. Is there any idea or suggestion for me?

- Inside my app
![Screen Shot 2022-01-05 at 10 41 18](https://user-images.githubusercontent.com/6580698/148153675-fe7a5843-8a61-4e3d-b9b1-7f0cf5194f47.png)

- Output image
![multixcumulativex0](https://user-images.githubusercontent.com/6580698/148153816-4bd64e76-30ff-4604-89b3-5d1c86f5f6bb.png)

**Comment by matinwd** [comment]
Having Same issue. Any ideas?
